sklearn/metrics/tests/test_classification.py

```diff
@@ -807,6 +807,7 @@ def test_precision_recall_f1_score_multilabel_2():
                         0.1666, 2)
 
 
+@ignore_warnings
```

Comment:
This test computes the precision/recall/f1_scores that are not defined in some cases, due to a zero division.
In these cases, the method returns 0 and sends a warning. I think these cases should be tested to see how the exception is handled, but we can silence the warnings.